ios/reactNativeApp-tvOS/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
```

Comment:
Can we remove this directory completely? We don't need `reactNativeApp-tvOS` right?

Reply:
I am 99% sure that we can remove this, I will test and remove it with the final PR